tests/wicked/before_test.pm

```diff
@@ -34,7 +34,7 @@ sub run {
     assert_script_run($enable_command_logging);
     # image which we using for sle15 don't have firewall running.
     # QAM need another way to figure out firewall state due to wider set of images
-    if (is_sle('<15') || (is_updates_tests() && systemctl("is-active " . opensusebasetest::firewall))) {
+    if (is_sle('<15') || (is_updates_tests() && !script_run('systemctl is-active -q ' . opensusebasetest::firewall))) {
```

Comment:
while this is correct I just want to mention alternatives:

1. Use `systemctl … , ignore_failure => 1`
2. Don't care about checking if the service is active when you disable it afterwards anyway, i.e.

```
systemctl('stop --now ' . opensusebasetest::firewall) if is_sle('<15') || is_updates_tests();
```

Reply:
Well, @asmorodskyi making the disabling conditional started the whole mess - so I really don't want to spend my weekends on anything fancy.